src/Core/Managed/Shared/TelemetryClient.cs

```diff
@@ -351,6 +351,12 @@ public void Track(ITelemetry telemetry)
                     return;
                 }
 
+                // If someone created configuration from scratch and forgot to initialize channel - use default
+                if (this.configuration.TelemetryChannel == null)
+                {
+                    this.configuration.TelemetryChannel = new InMemoryChannel();
+                }
+
```

Comment:
We had same initialization logic in CreateDefault and Active: if channel is not provied InMemory channel is used. Our policy is not to crash customer's application and provide defaults. That was not done in case customer decided to create TelemetryConfiguration from scratch so it fixes this behavior. Discussion was in the issue mentioned (#295)